smartsim/experiment.py

```diff
@@ -159,28 +161,63 @@ def __init__(
             exp_path = osp.join(getcwd(), name)
 
         self.exp_path = exp_path
+        """The path under which the experiment operate"""
 
-        self._launcher = launcher.lower()
-
-        if self._launcher == "auto":
-            self._launcher = detect_launcher()
-        if self._launcher == "cobalt":
-            raise SSUnsupportedError("Cobalt launcher is no longer supported.")
+        # TODO: Remove this! The controller is becoming obsolete
+        self._control = Controller(launcher="local")
 
-        if launcher == "dragon":
-            self._set_dragon_server_path()
+        self._active_launchers: set[LauncherProtocol[t.Any]] = set()
+        """The active launchers created, used, and reused by the experiment"""
 
-        self._control = Controller(launcher=self._launcher)
-
-        self.fs_identifiers: t.Set[str] = set()
+        self._fs_identifiers: t.Set[str] = set()
+        """Set of feature store identifiers currently in use by this
+        experiment"""
         self._telemetry_cfg = ExperimentTelemetryConfiguration()
+        """Switch to specify if telemetry data should be produced for this
+        experiment"""
+
+    def start_jobs(
```

Comment:
Should ``start_jobs`` replace ``start``?

Reply:
Yep! will rename when I remove `self._controller` and related methods!